packages/atlas/src/config/configSchema.ts

```diff
@@ -35,6 +35,7 @@ export const configSchema = z.object({
       googleOauthEndpoint: z.string().nullable(),
       youtubeSyncApiUrl: z.string().nullable(),
       youtubeCollaboratorMemberId: z.string().nullable(),
+      enrollmentReward: z.number(),
```

Comment:
I think this should be optional. I asked Dima about this. Will let you know once confirmed